db/patterns/kakao_tiara.eno

```diff
@@ -0,0 +1,20 @@
+name: Kakao Tiara
+category: site_analytics
+website_url:
```

Comment:
Are you aware of a website (owned by them) that describes the service?

Reply:
Tiara is their internal service, so they don't have any public stage. I agree to your opinion.